tasks/WhereIsThis.tex

```diff
@@ -1,19 +1,18 @@
 \section{Where is This? [Party Host]}
 \label{test:where-is-this}
-The robot has to explain to people where they can find places in the arena (e.g.~\emph{Where is the TV?}). The robot has either guide the operator there, or tell the operator how to get there (e.g.~\emph{It's at your right} or \emph{Turn around and go towards\dots}).
+The robot has to show people where they can find places in the arena (e.g.~\emph{Where is the TV?}). The robot has to tell the operator how to get there (in a summarized fashion) and then physically take the operator there in a type of tour guide.
```

Comment:
**explain AND show**. Just guiding people is too trivial for a Stage II test.

Reply:
Agreed. Changed.